src/main/java/net/glowstone/util/lang/LanguageProvider.java

```diff
@@ -12,7 +13,7 @@
     /**
      * Returns the internationalized message according to the effective locale of the player.
      */
-    String getString(String key, GlowPlayer p, Object ... args);
+    String getString(GlowPlayer p, String key, Object ... args);
```

Comment:
Please use `player` instead of `p`, and add documentation for each parameters + return value in Javadoc.